### PR TITLE
i18n(es): add styling page

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [x] New or updated content
- [ ] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

- Adds `guides/styling.md` in spanish
- Adds page to `nav.ts`

### CODE REVIEW

File: src/pages/es/guides/styling.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+```astro
+---
+// ¡Ejemplo avanzado! No recomendado para la mayoría de los usuarios.
+import rawStylesCSS from '../styles/main.css?raw';
+---
+<style is:inline set:html={rawStylesCSS}></style>
+```
+
+Consulte la [documentación de Vite](https://vitejs.dev/guide/assets.html#importing-asset-as-url) para obtener más detalles.
```
Comment (agustinmulet) [inline src/pages/es/guides/styling.md:277]:
```suggestion
Consulta la [documentación de Vite](https://vitejs.dev/guide/assets.html#importing-asset-as-url) para obtener más detalles.
```

File: src/pages/es/guides/styling.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+### `?url` Importaciones de CSS
+
+Para casos de uso avanzado, puedes importar una URL de referencia directa a un archivo CSS dentro de la carpeta `src/` de tu proyecto. Esto puede ser útil cuando necesitas un control completo sobre cómo se carga del CSS en la página. Sin embargo, esto evitará la optimización del CSS con el resto de la página.
+
+Esto no es recomendable para la mayoría de los usuarios. En su lugar, coloque los archivos CSS dentro de `public/` para obtener una ruta de URL.
```
Comment (agustinmulet) [inline src/pages/es/guides/styling.md:283]:
```suggestion
Esto no es recomendable para la mayoría de los usuarios. En su lugar, coloque los archivos CSS dentro de `public/` para obtener una ruta URL.
```

File: src/pages/es/guides/styling.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+
+:::caution
+Importar un archivo CSS más pequeño con `?url` puede devolver el contenido codificado en base64 como una URL de datos en la compilación final. Escriba su código para que sea compatible con datos codificados (`data:text/css;base64,...`) o configure [`vite.build.assetsInlineLimit`](https://vitejs.dev/config/#build-assetsinlinelimit) la opción de configuración a `0` para deshabilitar esta característica.
```
Comment (agustinmulet) [inline src/pages/es/guides/styling.md:286]:
```suggestion
Importar un archivo CSS pequeño con `?url` puede devolver el contenido codificado en base64 como una URL de datos en la compilación final. Escriba su código para que sea compatible con datos codificados (`data:text/css;base64,...`) o configure la opción [`vite.build.assetsInlineLimit`](https://vitejs.dev/config/#build-assetsinlinelimit) en `0` para deshabilitar esta característica.
```

File: src/pages/es/guides/styling.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+```astro
+---
+// ¡Ejemplo avanzado! No recomendado para la mayoría de los usuarios.
+import stylesUrl from '../styles/main.css?url';
+---
+<link rel="preload" href={stylesUrl} as="style">
+<link rel="stylesheet" href={stylesUrl}>
+```
+
+Consulte la [documentación de Vite](https://vitejs.dev/guide/assets.html#importing-asset-as-url) para obtener más detalles.
```
Comment (agustinmulet) [inline src/pages/es/guides/styling.md:298]:
```suggestion
Consulta la [documentación de Vite](https://vitejs.dev/guide/assets.html#importing-asset-as-url) para obtener más detalles.
```

**Review by agustinmulet — COMMENTED** [review]
Revisado, buen trabajo! 💯

**Comment by Yan-Thomas** [comment]
Thanks for the amazing job! Time to merge yay!

**Pull request merged**